frontend-react/src/hooks/UseLookupTable.ts

```diff
@@ -62,25 +78,43 @@ export async function getLatestData<T>(
     }
 }
 
+const getDataAndVersion = async <T>(
```

Comment:
this encapsulates some code that was being used by both getSenderAutomationData and getSenderAutomationDataRows